gr-digital/python/digital/__init__.py

```diff
@@ -23,20 +23,38 @@
     from .digital_python import *
 
 from gnuradio import analog  # just need analog for the enum
-from .psk import *
-from .qam import *
-from .qamlike import *
-from .bpsk import *
-from .qpsk import *
-from .gmsk import *
-from .gfsk import *
-from .cpm import *
+from .psk import (
+    create_encodings,
+    psk_mod,
+    psk_demod,
+    psk_constellation,
+)
+from .qam import(
+    is_power_of_four,
+    get_bit,
+    get_bits,
+    make_differential_constellation,
+    make_non_differential_constellation,
+    large_ampls_to_corners_mapping
+)
```

Comment:
Maybe for stuff like this we just do
```suggestion
from . import qam
```

...and then amend the API calls below. With calls like `get_bits()`, they are pretty ambiguously named, and it might  be nicer to not have them in the main namespace. But that would be an additional change over what you have already.